app/modules/scan/service.server.ts

```diff
@@ -138,3 +145,61 @@ export async function getScanByQrId({ qrId }: { qrId: string }) {
     });
   }
 }
+
+export async function createScanNote({ userId, qrId, latitude, longitude, manuallyGenerated }: {
+  userId?: string | null;
+  qrId: string;
+  latitude?: Scan["latitude"];
+  longitude?: Scan["longitude"];
+  manuallyGenerated?: boolean;
+}) {
+  try {
+    let message = "";
+    const { assetId, organizationId } = await getQr(qrId)
+    if (assetId) {
+      if (userId && userId != "anonymous") {
+        const user = await db.user.findFirst({
+          where: {
+            id: userId,
+          },
+          select: {
+            firstName: true,
+            lastName: true,
+          },
+        });
+
+        const userName = user?.firstName + " " + user?.lastName
+        if (manuallyGenerated) {
+          message = `**${userName}** manually updated the GPS coordinates to **${latitude} ${longitude}**`
```

Comment:
lat and long need to be separated by a coma. Also I think the bold is too much. Lets do italic instead.